index.js

```diff
@@ -131,9 +133,7 @@ window.addEventListener('click', function(e) {
             alert('Not posted yet.');
           }
           else {
-            destination = ipc.sendSync('saveEpisode', {
-              fileName: fileName
-            });
+            destination = dialog.showOpenDialog({properties: ['openDirectory'], title: 'Select folder to save episode to...'});
```

Comment:
break out the members of the params object into their own line like the old line did